@navikt/core/react/src/form/form-summary/FormSummary.tsx

```diff
@@ -0,0 +1,108 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef } from "react";
+import FormSummaryAnswer from "./FormSummaryAnswer";
+import FormSummaryAnswers from "./FormSummaryAnswers";
+import FormSummaryEdit from "./FormSummaryEdit";
+import FormSummaryHeader from "./FormSummaryHeader";
+import FormSummaryHeading from "./FormSummaryHeading";
+import FormSummaryLabel from "./FormSummaryLabel";
+import FormSummaryValue from "./FormSummaryValue";
+
+interface FormSummaryComponent
+  extends React.ForwardRefExoticComponent<
+    FormSummaryProps & React.RefAttributes<HTMLDivElement>
+  > {
+  /**
+   * Must include `<FormSummary.Heading>` and optionally `<FormSummary.Edit>`.
+   */
+  Header: typeof FormSummaryHeader;
+  /**
+   * Typographic Heading to use in the `FormSummary.Header` component.
+   */
+  Heading: typeof FormSummaryHeading;
+  /**
+   * Link to edit the answers to use in the `FormSummary.Header` component. Should link to the relevant part of the form.
+   */
+  Edit: typeof FormSummaryEdit;
+  /**
+   * Wrapper component for the answers.
+   */
+  Answers: typeof FormSummaryAnswers;
+  /**
+   * Wrapper component for each answer. To be used in the `FormSummary.Answers` component.
+   */
+  Answer: typeof FormSummaryAnswer;
+  /**
+   * Corresponds to the question in the form. To be used in the `FormSummary.Answer` component.
+   */
+  Label: typeof FormSummaryLabel;
+  /**
+   * Corresponds to the answer in the form. To be used in the `FormSummary.Answer` component.
+   */
+  Value: typeof FormSummaryValue;
+}
+
+export interface FormSummaryProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Must include:
+   *
+   *  - `<FormSummary.Header>`
+   *  - `<FormSummary.Answers>`
+   *
+   * @example
+   * <FormSummary>
+   *   <FormSummary.Header>
+   *    <FormSummary.Heading>HeadingTekst</FormSummary.Heading>
+   *    <FormSummary.Edit />
+   *   </FormSummary.Header>
+   *   <FormSummary.Answers>
+   *     <FormSummary.Answer>
+   *       <FormSummary.Label>Navn</FormSummary.Label>
+   *       <FormSummary.Value>Ola Nordmann</FormSummary.Value>
+   *     </FormSummary.Answer>
+   *   </FormSummary.Answers>
+   * </FormSummary>
+   */
+  children: React.ReactNode;
+}
+
+/**
+ * A summary of a previously answered form.
+ *
+ * @example
+ * <FormSummary>
+ *   <FormSummary.Header>
+ *    <FormSummary.Heading>HeadingTekst</FormSummary.Heading>
+ *    <FormSummary.Edit />
+ *   </FormSummary.Header>
+ *   <FormSummary.Answers>
+ *     <FormSummary.Answer>
+ *       <FormSummary.Label>Navn</FormSummary.Label>
+ *       <FormSummary.Value>Ola Nordmann</FormSummary.Value>
+ *     </FormSummary.Answer>
+ *   </FormSummary.Answers>
+ * </FormSummary>
+ */
+export const FormSummary = forwardRef<HTMLDivElement, FormSummaryProps>(
+  ({ children, className, ...rest }, ref) => {
+    return (
+      <section
```

Comment:
Siden `section` ikke har en aria-label eller `aria-labelledby` oppfører den seg som en `div` nå. Ikke 100%, men tror noen AXE-tester kan feile på at section ikke har label/beskrivelse. Må nesten verifisere med Morten her.

Reply:
Snakket med Morten. Han syntes det ikke var noen vits med `<section>` og at vi like godt kan bruke `<div>`.

Reply:
Time for the great <div>ening 😎 🎉 endrer til div.

Reply:
eller... lar det bare bli, var vel bare at det ikke gir noe _mer_ enn en div, ikke at det burde byttes til div 🤪

Reply:
Tenker at vi endrer til div da, i tilfelle Ken har rett i at AXE reagerer på section uten labe,

Reply:
Var ref denne: https://github.com/navikt/aksel/blob/9bbc3e68130793aad1a7cb1eae785590671e1a50/aksel.nav.no/website/e2e/axe.e2e.ts#L27
(ikke så mye context å lese ut fra den linjen da...)